Assets/_BForBoss/_Character/Scripts/FirstPersonPlayer.cs

```diff
@@ -263,12 +247,7 @@ private void ToggleThirdPerson()
             cmThirdPersonCamera.gameObject.SetActive(IsThirdPerson);
             TogglePlayerModel();
         }
-
-        private void SwitchView(InputAction.CallbackContext context)
-        {
-            if (context.started) IsThirdPerson = !IsThirdPerson;
-        }
-
+        
         private void TogglePlayerModel()
         {
             animate = IsThirdPerson;
```

Comment:
It might make sense to move the toggle player model code into FirstPersonPlayer.ThirdPerson, what are your thoughts?

Reply:
sure thing, i just didn't wanna break ur pr